### DISCUSSION TITLE
Understanding IFCtoLBD converter

### DISCUSSION THREAD

**Discussion by sumwas** [jyrkioraskari/IFCtoLBD#82] [Q&A]
Hello, I am a student reasearcher and would like to ask a few questions regarding this convertor. 

1. When using the converter you can generate a RDF file with or without using ifcOWL. The one without ifcOWL uses the bot ontology to classify ifc elements while the file using ifcOWL doesn't seem to use the bot ontology at all. Can these files be queried together? And if not, is there a way to combine these two files to create a larger dataset to query? 
2. Is the ifc ontology this converter uses the orginal ifc ontology or a modified version for LBD?
3. Why does the bot ontology not use specific type information? How is one supposed to distinguish a door element from a window element without having to look at its unique ID name? 

Thank you!

**Comment by jyrkioraskari** [comment]
Hi sumwas,

1.	 Yes, they can be queried together. When you select the “Create and link ifcOWL” option, the converter adds  owl:sameAs links that points to the equivalent ifcOWL elements.
2.	It uses the LBD ontologies. 
3.	The elements that can be associated with the Building Element Ontology, are given the more specific type.
Kind regards,

J
